libr/debug/p/native/linux/linux_debug.c

```diff
@@ -1180,357 +1193,357 @@ static linux_map_entry_t *linux_get_mapped_files(RDebug *dbg, ut8 filter_flags)
 			pmentry->dumpeable = 1;
 		}
 			
-                ADD_MAP_NODE (pmentry);
-                p = strtok_r (NULL, "\n", &end_line);
-        }
+		ADD_MAP_NODE (pmentry);
+		p = strtok_r (NULL, "\n", &end_line);
+	}
 
-        mapping_file.size += sizeof (unsigned long) * 2; /* number of mappings and page size */
-        eprintf ("mapping_file.size: %d\n", mapping_file.size);
+	mapping_file.size += sizeof (unsigned long) * 2; /* number of mappings and page size */
```

Comment:
unsigned long changes between 32 and 64bits. so this code is not portable. use sizeof(ut32) or sizeof(ut64)